### DISCUSSION TITLE
Welcome to Zim Discussions!

### DISCUSSION THREAD

**Discussion by ericbn** [zimfw/zimfw#420] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Eriner** [comment]
I'm excited for this GitHub feature. Long overdue.

**Comment by antoineco** [comment]
I started using Zim in 2017 due to concerns whether or not Prezto was still going to be maintained (https://github.com/sorin-ionescu/prezto/issues/1239), and never felt like moving back (\*). You folks have managed to keep the quality high over these years, and always stayed faithful to your promise of keeping Zim fast and minimal.

I work almost exclusively in a terminal, and Zim is a real game changer. Thank you! 🙌 

(\*) Even though Prezto is also a high-quality framework.
